pkg/rsnotify/listener/listener_test.go

```diff
@@ -3,13 +3,9 @@ package listener
 // Copyright (C) 2022 by RStudio, PBC.
 
 import (
-	"testing"
-
 	"gopkg.in/check.v1"
 )
 
-type NotifySuite struct{}
-
-var _ = check.Suite(&NotifySuite{})
+type ListenerSuite struct{}
 
-func TestPackage(t *testing.T) { check.TestingT(t) }
```

Comment:
Removing this is causing the build to fail. You need one call like this per package. You can either revert this one or move it to `genericmatcher_test.go`.